src/buildvars.py

```diff
@@ -99,20 +99,22 @@ def _fix_single_ec2_node(stackname):
 
     stack_all_ec2_nodes(stackname, (_fix_single_ec2_node, {'stackname': stackname}), username=BOOTSTRAP_USER)
 
-# TODO: deletion candidate. can only ever do a shallow update
 @requires_aws_stack
 def force(stackname, field, value):
-    "replace a specific key with a new value in the buildvars for all ec2 instances in stack"
+    """replace a specific key with a new value in the buildvars for all ec2 instances in stack.
```

Comment:
"replaces value at 'field' with given `value` for all buildvars on all ec2 instances on given `stackname`."